<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExcelExportGUIClass</class>
 <widget class="QMainWindow" name="ExcelExportGUIClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1021</width>
    <height>673</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ExcelExportGUI</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPlainTextEdit" name="plainTextEdit">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>90</y>
      <width>1021</width>
      <height>531</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>日志输出</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnPath">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>选择数据目录</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtPath">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>10</y>
      <width>371</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnWork">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>开始处理</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtYear">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>10</y>
      <width>271</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>60</y>
      <width>391</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>注意：有剩余的缺失的数据缺少太多并且格式不对 放弃</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1021</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="ExcelExportGUI.qrc"/>
 </resources>
 <connections/>
</ui>
